README.md

```diff
@@ -20,24 +20,11 @@ Currently tested in:
 * Firefox 18
 * Safari 5.1.1
 
-Copyright and License
----------------------
-
-Copyright 2019 Mark Rolich & Frank Schmidt & Agnes Köhler
```

Comment:
Should we add our names and update the date in the license.txt?

Reply:
Will do that 👍